github/Organization.py

```diff
@@ -726,6 +760,19 @@ def has_in_public_members(self, public_member):
         )
         return status == 204
 
+    def remove_from_membership(self, member):
+        """
+        :calls: `DELETE /orgs/:org/memberships/:user <http://developer.github.com/v3/orgs/memberships>`_
+        :param member: :class:`github.NamedUser.NamedUser`
+        :rtype: None
+        """
+        assert isinstance(member, github.NamedUser.NamedUser), member
```

Comment:
What exactly is the difference between `remove_from_members` and `remove_from_membership`?

If I understand correctly, the only difference is that `remove_from_membership`  is for non-active members, in which case this method will cancel the invitation?

Reply:
Correct. Per the [docs](https://developer.github.com/v3/orgs/members/#remove-organization-membership):

```
If the specified user is an active member of the organization, this will remove them from
the organization. If the specified user has been invited to the organization, this will cancel 
their invitation. The specified user will receive an email notification in both cases.
```